backend/src/main/java/com/google/sps/servlets/FindRecipesServlet.java

```diff
@@ -48,7 +48,7 @@ protected void get(HttpServletRequest request, HttpServletResponse response) thr
 
   @Override
   protected void post(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String ingredients = request.getReader().readLine();
+    String ingredients = request.getReader().readLine().replaceAll(" ", "%20");
```

Comment:
Custom search API kept failing because of the spaces in people's ingredients, i.e. "chicken breast"